kong/clustering/data_plane.lua

```diff
@@ -101,7 +101,7 @@ local function send_ping(c, log_suffix)
             "unable to send ping frame to control plane: ", err, log_suffix)
 
   else
-    ngx_log(ngx_DEBUG, _log_prefix, "sent ping frame to control plane", log_suffix)
+    ngx_log(ngx_DEBUG, _log_prefix, "sent ping frame to control plane with hash: ", hash, log_suffix)
```

Comment:
should we only log this once per hash and not per ping.

Reply:
just to confirm you mean we could log the new hash only after we process the reconfigure event and update the declarative config? in that case, yes we could do that as well.
but this log line also gives us a good idea of the dataplane's configuration state. furthermore, it's not really adding a new log statement but merely appending a few bytes to an existing log statement.

Reply:
@bungle updated accordingly.